docs/web/user_guide.md

```diff
@@ -600,14 +600,31 @@ filter arguments:
   --report-hash [REPORT_HASH [REPORT_HASH ...]]
                         Filter results by report hashes.
   --review-status [REVIEW_STATUS [REVIEW_STATUS ...]]
-                        Filter results by review statuses. This can be used
-                        only if basename or newname is a run name (on the
-                        remote server). (default: ['unreviewed', 'confirmed'])
+                        Filter results by review statuses.
+                        Reports can be assigned a review status of the
+                        following values:
+                        - Unreviewed (default): Nobody has seen this report.
```

Comment:
I think the `default` here and the `default` filter value (which contains the confirmed value too) a few lines below can be a bit confusing.
How about removing the default word from here or to extend the description a bit? (`default value for each report` or something like this)